rpc/master/service_server.go

```diff
@@ -13,7 +13,31 @@
 
 package master
 
-import "github.com/control-center/serviced/domain/service"
+import (
+	"time"
+
+	"github.com/control-center/serviced/domain/service"
+)
+
+type ServiceUseRequest struct {
+	ServiceID   string
+	ImageID     string
+	ReplaceImgs []string
+	Registry    string
+	NoOp        bool
+}
+
+type WaitServiceRequest struct {
+	ServiceIDs []string
+	State      service.DesiredState
+	Timeout    time.Duration
+	Recursive  bool
+}
+
+type EvaluateServiceRequest struct {
```

Comment:
Yes.

I guess I waffled on that a bit. Should we have unique xyzRequest object for each request (even ones that have the same inputs), or should we share a common object for different calls which need the same arguments. When I created EvaluateServiceRequest I was thinking the former, but I guess I got a little lazy as I got deeper into the refactor and opted for the later.

Thoughts?